src/agent/config/operation.py

```diff
@@ -109,15 +109,16 @@ def get_settings(self) -> dict:
         Gets the settings of the operation
         :return: a dict representing the operations
         """
-        super().get_settings()["headless"] = self.headless
-        return super().get_settings()
+        settings = super().get_settings()
+        settings["headless"] = self.headless
+        return settings
```

Comment:
why did you change this?

Reply:
it wasn't working with how it was originally. doing this:
super().get_settings()["headless"] = self.headless
        return super().get_settings()
was returning an empty string